example-singleServer-noBackend.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>HTML5 Speedtest</title>
+<meta name="viewport" content="width=device-width">
+<link rel="shortcut icon" href="favicon.ico">
+<script src="speedtest.js"></script>
+<script>
+function I(i){return document.getElementById(i);}
+
+//INITIALIZE SPEEDTEST
+var s=new Speedtest(); // create speedtest object
+
+//CUSTOM SETTINGS for version with no PHP or other server-side scripting
+
+// we only want ping, download and upload test
+s.setParameter("test_order", "P_D_U");
+// disable telemetry
+s.setParameter("telemetry_level", "0");
+
+// path to a large static file, used for download test.
+//    to create this file, execute the following command:
+//    dd if=/dev/zero of=10mb.dat bs=10485760 count=1
```

Comment:
Data needs to be incompressible. You should replace this with /dev/urandom

Reply:
I changed that, too. Binary files (*.dat) are not compressed by the server by default.